### ISSUE TITLE
trigger reupload of globally pinned content

### ISSUE THREAD

**Issue by zelig** [ethersphere/bee#891]
# User story

## Story

As a user, when downloading globally pinned content, I want to limit the number of recovery requests  sent to targets. After a number of missing chunks, I expect the recovery target nodes (global pinners) to reupload the entire content. see also #852 

As a global pinner, I want to detect if a number of chunks are requested as missing that are part of the same pinned content so that i can reupload the entire file/collection assuming more chunks are lost. 

## Background

Basically global pinners need to get information about the context of the missing chunk, ie, the pinned root hash as part of the recovery request. All this without leaking information about which chunk belong to which root. 
This is easy to do based on this architecture proposal https://hackmd.io/9LhG0OpGSgyzC3QBZN_EwA?view

## Tasks

| Task | Assignee |  Done (1), or 0 |
| ------ |------------| -----|

**Comment by Eknir** [comment]
Thanks for making this issue @zelig !
